src/components/DetailsButton/DetailsButton.test.tsx

```diff
@@ -0,0 +1,17 @@
+import { BrowserRouter as Router } from "react-router-dom";
+import { render, screen } from "@testing-library/react";
+import { DetailsButton } from ".";
+
+describe("DetailsButton tests:", () => {
+  test("should render DetailsButton component", () => {
+    render(
+      <Router>
```

Comment:
you don't use the router there - I think you need another test - to click this button an check current route

Reply:
This component needs Router to work, otherwise Jest shows an error.